### ISSUE TITLE
Conflicts with VoiceAttack app

### ISSUE THREAD

**Issue by DonSmashio** [chalkos/RepairMe#13]
RepairMe conflicts with https://store.steampowered.com/app/583010/VoiceAttack/ VoiceAttack. It prevents VoiceAttack from sending commands into the game. Instead, it sends the commands -somewhere-.

I don't think this happened before. It started happening today.

RepairMe also adds something on the top left of the screen. I was able to pull out a notification but there is still something up there.

https://imgur.com/a/1FIOFGj See the strange thing on the top left.

Disabling RepairMe allows VoiceAttack to work properly and removes the thing in the top left.

https://imgur.com/a/B3UANmH

Image above shows the VoiceAttack config window. If I set target to FINAL FANTASY XIV while RepairMe is active, it won't send anything to FF14, instead, it sends it to a different "window" that doesn't exist. It unfocuses the game.

**Comment by chalkos** [comment]
Lets try a few things to see what is happening.

Regarding the top left of your screen, have you changed your game resolution recently? changing it may put RepairMe's widgets in weird spots.

If you disable all bars, percentages and alerts in repairme config one by one, can you identify which one is showing outside of your screen? (note that there are settings for condition and spiritbond, from the picture I'd say that is the spiritbond bar)

Another thing you can try is to make it sure all enabled RepairMe widgets are on screen (repairconfig, then enable "Move UI" and drag them) and then try the Voiceattack thing again. Does it work?

When any plugin/dalamud window is outside the game are it is handled in a slightly different way by ImGui (the framework Dalamud uses to show windows) so maybe it is creating a separate space to which VoiceAttack is sending the commands. While VoiceAttack is working (disable RepairMe if you need to), can you try doing `/xllog` and drag the log window to be partly outside the game window (its background should become opaque when you do), and try VoiceAttack again? Does it still work?
It might not be fixable or not in the near future, but at least we'll know about this problem and what causes it.

**Comment by DonSmashio** [comment]
Turns out it was "enable multi-monitor support" that did it.

What happened was that my audio had broken down and I used the suggestion to switch to DX9 to fix it. It fixed it but also disabled borderless mode and changed the resolution. I later changed back to DX11.

This resulted in the repairme spiritbond and condition percentages widgets to get stuck inside the corner of the game window. They are quite small and I suppose that this prevents them from being clickable when up in that corner.

Turning off multi-monitor support nudged them 1 pixel into the game window and made them clickable again.

I had the same problem when I accidentally dragged the ping plugin widget out of the game window. Ping plugin has the ability to reset the widget locations so I tried that and VoiceAttack worked again.

I managed to fix the RepairMe issue but may I suggest having a "reset all widget locations" option?
